src/main/java/uk/gov/cshr/civilservant/repository/CivilServantRepository.java

```diff
@@ -11,6 +11,7 @@
 import uk.gov.cshr.civilservant.domain.*;
 import uk.gov.cshr.civilservant.dto.CivilServantReportDto;
 
+import java.util.Arrays;
```

Comment:
I can't see the use of this import. Please remove redundant imports if not used.

Reply:
Done. Will be in next commit.